eslint.config.js

```
@@ -53,4 +53,23 @@ module.exports = defineConfig(
     },
   },
+  {
+    files: ['playwright/e2e/**/*.ts', 'playwright/e2e/**/*.tsx'],
+    languageOptions: {
+      parser: tsParser,
+    },
+    rules: {
+      'no-restricted-syntax': [
+        'error',
+        {
+          selector: 'MemberExpression[object.name="page"][property.name="route"]',
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)